features/smil-svg-animations.yml.dist

```diff
@@ -1,18 +1,18 @@
-# Generated from: svg-animations.yml
+# Generated from: smil-svg-animations.yml
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
   baseline: high
-  baseline_low_date: 2020-01-15
-  baseline_high_date: 2022-07-15
+  baseline_low_date: ≤2020-01-15
+  baseline_high_date: ≤2022-07-15
```

Comment:
I sent the (non-blocking) https://github.com/mdn/browser-compat-data/pull/24607 to remove the ranges. It shouldn't change the overall date, just remove the uncertainty.